src/envoy/http/authn/authenticator_base.cc

```diff
@@ -71,17 +72,59 @@ void AuthenticatorBase::validateX509(
     done_callback(&payload, false);
   }
 
-  // TODO (lei-tang): Adding other attributes (i.e ip) to payload if desire.
+  // TODO (lei-tang): Adding other attributes (i.e ip) to payload if needed.
   done_callback(&payload, true);
 }
 
+// Build JWT attributes from JWT payload
+// TODO (lei-tang): Build more attributes/claims to payload if needed.
+void buildJwtAttributes(JwtPayload* payload) {
+  ::google::protobuf::Map< ::std::string, ::std::string>& claims =
+      *payload->mutable_claims();
```

Comment:
why do you need mutable_claims?   you are NOT modifying it.

Reply:
This function has been removed.